hapi-fhir-sql-migrate/src/test/java/ca/uhn/fhir/jpa/migrate/HapiMigratorIT.java

```diff
@@ -61,13 +68,27 @@ void before() {
 	@AfterEach
 	void after() {
 		myJdbcTemplate.execute("DROP TABLE " + MIGRATION_TABLENAME);
+		myJdbcTemplate.execute("DROP TABLE IF EXISTS " + TABLE_NAME_HFJ_RES_REINDEX_JOB);
+		myJdbcTemplate.execute("DROP TABLE IF EXISTS " + TABLE_NAME_NON_SCHEMA_INIT);
+		myJdbcTemplate.execute("DROP TABLE IF EXISTS " + TABLE_NAME_SCHEMA_INIT);
+		myJdbcTemplate.execute("DROP TABLE IF EXISTS " + TABLE_NAME_NON_SCHEMA_INIT_2);
+		myJdbcTemplate.execute("DROP TABLE IF EXISTS " + TABLE_NAME_SCHEMA_INIT_2);
 		assertEquals(0, myDataSource.getNumActive());
 		HapiMigrationLock.setMaxRetryAttempts(HapiMigrationLock.DEFAULT_MAX_RETRY_ATTEMPTS);
 		System.clearProperty(HapiMigrationLock.CLEAR_LOCK_TABLE_WITH_DESCRIPTION);
 	}
 
-	@Test
-	public void testInitializeSchema() {
+	@ParameterizedTest
```

Comment:
minor: parameterized booleans are rough to read.  Maybe leave some javadoc explaining the purpose.
```suggestion
/** We test initialization in two cases: an empty database, and one that has been manually filled by running the schema sql. */
		@ParameterizedTest
```